### PR TITLE
Kstream for cstats

### PR DESCRIPTION
**Pull request by mazhurin**

### CODE REVIEW

File: deployment/kafka_stream/src/main/java/ie/equalit/baskerville/streams/stats/model/Weblog.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        this.VIEWED_PAGE_COUNT = VIEWED_PAGE_COUNT;
+        this.UA = UA;
+        this.HTTP_CODE = HTTP_CODE;
+        this.WINDOW_END = WINDOW_END;
+    }
+
+    @Override
+    public String toString() {
+        return "WeblogStat {" +
+        "host='" + this.HOST  +
```
Comment (mhqz) [inline deployment/kafka_stream/src/main/java/ie/equalit/baskerville/streams/stats/model/Weblog.java:56]: hi @mazhurin, this is probably a minor thing but I'm not sure that single quotes are properly paired in the `toString` methods of the Java models.

File: deployment/kafka_stream/src/main/java/ie/equalit/baskerville/streams/stats/serde/JsonSerializer.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    @Override
+    public byte[] serialize(String topic, T t) {
+        return gson.toJson(t).getBytes(Charset.forName("UTF-8"));
+    }
+
+    @Override
+    public void close() {
+
+    }
+}
```
Comment (mhqz) [inline deployment/kafka_stream/src/main/java/ie/equalit/baskerville/streams/stats/serde/JsonSerializer.java]: We can probably add some newline at the end of the new files so we can avoid complaints from diff.

File: deployment/kafka_stream/src/main/java/ie/equalit/baskerville/streams/stats/model/Weblog.java (hunk 1 above)
Comment (mazhurin) [reply deployment/kafka_stream/src/main/java/ie/equalit/baskerville/streams/stats/model/Weblog.java:56]: It should work. The compiler will create a `String` object on the stack. I just did a simple test to confirm:
```
public class Kuku {

    public static void main(String[] args) throws Exception {

        Kuku a = new Kuku();
        System.out.println(a.get_string());
        System.out.println(a.get_string().getClass());
    }

    public String get_string(){
        return "a" + "b";
    }
}
```
________
ab
class java.lang.String

Comment (mazhurin) [reply deployment/kafka_stream/src/main/java/ie/equalit/baskerville/streams/stats/model/Weblog.java:56]: done

File: deployment/kafka_stream/src/main/java/ie/equalit/baskerville/streams/stats/serde/JsonSerializer.java (hunk 2 above)
Comment (mazhurin) [reply deployment/kafka_stream/src/main/java/ie/equalit/baskerville/streams/stats/serde/JsonSerializer.java]: done

File: deployment/kafka_stream/src/main/java/ie/equalit/baskerville/streams/stats/model/Weblog.java (hunk 1 above)
Comment (mhqz) [reply deployment/kafka_stream/src/main/java/ie/equalit/baskerville/streams/stats/model/Weblog.java:56]: OK, thanks for the clarification :+1:

**Review by mhqz — APPROVED** [review]

**Pull request merged**